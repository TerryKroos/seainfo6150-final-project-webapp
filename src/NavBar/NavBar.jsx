import React from 'react';
import {BrowserRouter as Router, Link} from 'react-router-dom';
import styles from './NavBar.module.css';

const NavBar = () => {
    return(
        <nav>
           <ul>
              <li >The Sports Blog</li>
              <li>
                  <Router>
                  <Link className={styles.link} to="/">
                 HOME</Link>
                  </Router>
              </li>
              <li>
              <Router>
                  <Link className={styles.link} to="/nba">
                      NBA
                 </Link>
                  </Router>
              </li>
              <li>
              <Router>
                  <Link className={styles.link} to="/nfl">
                 NFL</Link>
                  </Router>
              </li>
              <li>
              <Router>
                  <Link className={styles.link} to="/nhl">
                 NHL</Link>
                  </Router>
              </li>
              <li>
              <Router>
                  <Link className={styles.link} to="/mbl">
                 MBL</Link>
                  </Router>
              </li>
              <li>
              <Router>
                  <Link className={styles.link} to="/soccer">
                 SOCCER</Link>
                  </Router>
              </li>
           </ul>
        </nav>
        )
}

export default NavBar;