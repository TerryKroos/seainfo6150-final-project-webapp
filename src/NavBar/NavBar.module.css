@import url('https://fonts.googleapis.com/css2?family=Inter:wght@200;700&display=swap');
  
  nav {
      margin-top: 25px;
      font-size: 18px;
      background-color: #f1f1f1;
      color: #556575;
      font-family: 'Inter', sans-serif;
  }
  

  nav ul {
      list-style-type: none;
      margin: 0;
      padding: 0;
      display: flex;
      justify-content: flex-start;
  }
  
  nav ul li {
      margin-left: 20px;
      padding: 0.5rem;
      text-align: center;
      text-decoration: none;

  }
  .link{
    text-decoration: none;
    color: #556575;
    cursor: pointer;
  }
  .link:hover{
    color: #282828; 
  }


  
  /* Responsive Nav Bar */
  @media all and (max-width: 600px) {
      nav ul {
          flex-direction: column;
      }
      nav ul li {
          text-align: center;
      }
  }
  