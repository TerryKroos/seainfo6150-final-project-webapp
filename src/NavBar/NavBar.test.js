
import React from "react";
import NavBar from "./NavBar";
import { render } from "@testing-library/react";

describe("NavBar tests", () => {
    it("renders correctly", () => {
      const { container } = render(<NavBar />);
      expect(container).toMatchSnapshot();
    });
  
  });