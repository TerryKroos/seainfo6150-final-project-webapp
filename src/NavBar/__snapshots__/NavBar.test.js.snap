// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar tests renders correctly 1`] = `
<div>
  <nav>
    <ul>
      <li>
        The Sports Blog
      </li>
      <li>
        <a
          class="link"
          href="/"
        >
          HOME
        </a>
      </li>
      <li>
        <a
          class="link"
          href="/nba"
        >
          NBA
        </a>
      </li>
      <li>
        <a
          class="link"
          href="/nfl"
        >
          NFL
        </a>
      </li>
      <li>
        <a
          class="link"
          href="/nhl"
        >
          NHL
        </a>
      </li>
      <li>
        <a
          class="link"
          href="/mbl"
        >
          MBL
        </a>
      </li>
      <li>
        <a
          class="link"
          href="/soccer"
        >
          SOCCER
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
